tutor/resources/styles/components/task-plan/controls.less

```diff
@@ -8,6 +8,11 @@
     font-weight: 600;
     border-radius: 3px;
 
+    &.preview-btn {
+        border-radius: 4px;
+        font-weight: normal;
```

Comment:
there are 4 spaces here instead of the 2 we usually use.  otherwise looks good.

although i wonder whether in this case, if we should add classes to the other buttons that the `font-weight: 600;` since those are kinda the weird buttons compared to our normal button styles, should apply to, but that is more code change...oh hmm...are those other builder footer buttons supposed to be bolded in the first place @Fredasaurus ?